src/Utility/WPCLIMigrationGTIN.php

```diff
@@ -211,4 +210,25 @@ protected function process_items( array $items ): int {
 	public static function is_needed(): bool {
 		return defined( 'WP_CLI' ) && WP_CLI;
 	}
+
+	/**
+	 * Add some info in the debug console.
+	 * Add --debug to see this kind of logs in WP CLI
```

Comment:
Just a small nitpick about the wording (this should be plural):
```suggestion
	 * Add --debug to see these logs in WP CLI
```